File: examples/id/generate/go/generateASnowflakeId.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/id"
)

// Generate a unique ID. Defaults to uuid.
func GenerateAsnowflakeId() {
	idService := id.NewIdService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := idService.Generate(&id.GenerateRequest{
		Type: "snowflake",
	})
	fmt.Println(rsp, err)
}

File: examples/id/generate/node/generateASnowflakeId.js
const { IdService } = require("micro-js-client/id");

// Generate a unique ID. Defaults to uuid.
async function generateAsnowflakeId() {
  let idService = new IdService(process.env.MICRO_API_TOKEN);
  let rsp = await idService.generate({
    type: "snowflake",
  });
  console.log(rsp);
}

generateAsnowflakeId();
